src/Routes.tsx

```diff
@@ -57,159 +62,239 @@ export interface ChildProps {
     technicalSchemas: object[];
     economicSchemas: object[];
     hubSchemas: object[];
-  }
+  };
   coaches: any[];
 }
 
 export interface RouteWithChildProps extends RouteProps {
   childProps: ChildProps;
 }
 
-export default ({ childProps }: RouteWithChildProps) => (
+export default ({ childProps, ...rest }: RouteWithChildProps) => (
```

Comment:
What is this ..rest? Is this a new React Router 6 thing?

Reply:
Honestly, I'm not sure if its needed or not, I couldnt get something to work so I was experimenting since you had {...rest} in all of the AppliedRoute, AuthenticatedRoute, and UnauthenticatedRoute. wrapped component. It has nothing to do with RR6.

Reply:
No, it is some props that are in app.tsx, experiencemodule, and learndashboard. They were a part of the 3 wrapped router components that cant be used in RR6 any more, so I just took it out of them and put them in each route.

Reply:
Cool